### ISSUE TITLE
Downwards keyboard navigation doesn't work if selected file is at right edge of view.

### ISSUE THREAD

**Issue by uncertainquark** [elementary/files#958]
If I am keyboard navigating files in a folder like this:

![Screenshot from 2019-06-03 08 54 05](https://user-images.githubusercontent.com/22585732/58774015-2cf48600-85dd-11e9-99a3-53ebeb6dead0.png)

Then the "Down arrow" keyboard button doesn't work if there is no file below the selected file i.e. it doesn't navigate below its existing position.

A similar fix was recently applied to the Applications Menu, albeit with horizontal scrolling.
